.github/workflows/ccpp.yml

```diff
@@ -0,0 +1,27 @@
+name: Test
+on:
+  pull_request:
+    branches:
+      - multinode
```

Comment:
Branch `multinode` doesn't exist in this repo. Should this PR be against public master instead?

Reply:
yes, forgot to update. Thanks for catching.